OCABundles/schema/bcgov-digital-trust/member-card-showcase/OCABundle.json

```diff
@@ -9,14 +9,8 @@
         "Surname": "Text"
       },
       "classification": "",
-      "digest": "EudfUE3gTdDatjnTOVQDFLC3hgEm3t5MWPY-BseiGdv8",
-      "flagged_attributes": [
-        "PPID",
-        "Member Status Code",
-        "Surname",
```

Comment:
Similar comment — Surname and Given Name is likely PII, even in the professional context of a lawyer.